src/presenters/addressPresenter.test.js

```diff
@@ -5,58 +5,116 @@ const {
 } = require("./addressPresenter");
 
 describe("Address Presenter", () => {
-  describe("generateSearchResultString", () => {
-    const data = [
-      {
-        address: {
-          organisationName: "My company",
-          departmentName: "My deparment",
-          buildingName: "my building",
-          subBuildingName: "Room 5",
-          buildingNumber: "1",
-          dependentStreetName: "My outter street",
-          streetName: "my inner street",
-          doubleDependentAddressLocality: "My double dependant town",
-          dependentAddressLocality: "my dependant town",
-          addressLocality: "my town",
-          postalCode: "myCode",
+  context("generateSearchResultString", () => {
+    it("should generate search result string with street name", () => {
+      const data = [
+        {
+          address: {
+            organisationName: "My company",
+            departmentName: "My deparment",
+            buildingName: "my building",
+            subBuildingName: "Room 5",
+            buildingNumber: "1",
+            dependentStreetName: "My outter street",
+            streetName: "my inner street",
+            doubleDependentAddressLocality: "My double dependant town",
+            dependentAddressLocality: "my dependant town",
+            addressLocality: "my town",
+            postalCode: "myCode",
+          },
+          text: "My company My deparment my building Room 5 1 My outter street my inner street, My double dependant town my dependant town my town, myCode",
         },
-        text: "My company My deparment my building Room 5 1 My outter street my inner street, My double dependant town my dependant town my town, myCode",
-      },
-    ];
+      ];
 
-    data.forEach((addressData, index) => {
-      it(`should match test address ${index} to its expected text output`, () => {
-        const output = generateSearchResultString(addressData.address);
-        expect(output).to.equal(addressData.text);
+      data.forEach((addressData, index) => {
```

Comment:
I'm not usually a fan of looping over tests, but I think they are useful here.

I can see a refactoring where the `data` object has an additional property `description`.

This would mean that you could have all your tests in an array, and a single test worded more like ...
```
data.forEach((addressData, index) => {
  it('should generate search result ${addressData.description},() => {
    const output = generateSearchResultString(addressData.address);
    
    expect(output).to.equal(addressData.text);
  });
});
```

No need to hold up this PR for it, but a useful refactoring to consider.